mongodb/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mongo:5.0.5
+
+# Create keyfile
+RUN openssl rand -base64 756 > /mongo-keyfile
```

Comment:
"Keyfiles are bare-minimum forms of security and are best suited for testing or development environments. For production environments we recommend using x.509 certificates." - https://www.mongodb.com/docs/manual/tutorial/enforce-keyfile-access-control-in-existing-replica-set/

Is `docker-compose.yml` our prod server?

Reply:
Also - Is this creating the mongo keyfile on the docker image itself -- is this safe in terms of secret management?

Reply:
This is purely for local dev yes - it's the keyfile that is used for the different mongodb instances in a replica set to talk to each other. When deployed azure takes care of all the replica set/keyfile issues for us and we don't need to worry about it :) It is safe as it's essentially throwaway in our local environment